loops/csharp/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿var u = int.Parse(args[0]);
+var r = Random.Shared.Next(0, 10_000);
+var a = new int[10_000];
+
+for (var i = 0; i < 10_000; i++)
+{
+    for (int j = 0; j < 10_000; j++)
```

Comment:
I believe line 7 should be 100_000 instead of 10_000
(checked from Rust code)

Reply:
thanks! I didn't saw this, i'll fix it :)